reverse_words/__init__.py

```diff
@@ -243,6 +244,7 @@ def main(mode, save_path, num_batches, data_path=None):
                 # Saving the model and the log separately is convenient,
                 # because loading the whole pickle takes quite some time.
                 Checkpoint(save_path, every_n_batches=500,
+                           parameters=model.parameters,
```

Comment:
Do we need this? I though `model.parameters` can be requested automatically by `Checkpoint`.